src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -280,6 +282,33 @@ public function setRequiredFieldsAction(Request $request)
         return $this->redirectToRoute('admin_customers_index');
     }
 
+    /**
+     * Search for customers by query.
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function searchAction(Request $request)
+    {
```

Comment:
We need some `AdminSecurity` here, we cannot allow anyone logged in the BO to access this search function.

Reply:
good catch. :+1: